README.md

```diff
@@ -1,33 +1,66 @@
-# tenableio2securityhub
-A python script for sending vulnerabilities from Tenable.io to AWS SecurityHub.
+# Tenable.io -> AWS Security Hub Transformer
 
-### Installation
-```
-pip install boto3 requests
-```
-
-### Python Version
-3.7+
-
-### Configuration
-In order to use this script, you must first Subscribe to Tenable.io as a provider in AWS SecurityHub.
-
-Once subscribed, update the following setting in the script with your [AWS Account ID](https://docs.aws.amazon.com/IAM/latest/UserGuide/console_account-alias.html#FindingYourAWSId):
+This tool is designed to consume Tenable.io asset and vulnerability data,
+transform that data into the AWS Security Hub Finding format, and then upload
+the resulting data into AWS Security Hub.
 
-    AWS_ACCOUNT_ID = '<YOUR_ACCOUNT_ID>'
+The tool can be run as either a one-shot docker container or as a command-line
+tool.  To run as a docker image, you'll need to build the image and then pass
+the necessary secrets on to the container.
 
-Next, update the following setting with your [AWS Region](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html):
+To run as a command-line tool, you'd need to install the required python modules
+and then can run the tool using either environment variables or by passing the
+required parameters as run-time parameters.
 
-    AWS_REGION = '<YOUR_AWS_REGION>'
+### Building for Docker
 
-Then, update the following settings with your [Tenable.io](https://cloud.tenable.com/) API keys:
+```shell
+docker build -t tio2sechub:latest .
+```
 
-    ACCESS_KEY = '<YOUR_ACCESS_KEY>'
-    SECRET_KEY = '<YOUR_SECRET_KEY>'
+### Installing Python Requirements
+```shell
+pip install -r requirements.txt
+```
 
```

Comment:
Note, I received the message below upon first run.  We should probably list minimum pip version of 18.1

You are using pip version 10.0.1, however version 18.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command

Reply:
honestly pip versions shouldn't matter much and are generally outside the scope of this code.